src/testdir/test_autocmd.vim

```diff
@@ -4537,13 +4537,13 @@ func Test_Changed_ChangedI()
     call term_sendkeys(buf, "\<Esc>")
     call WaitForAssert({-> assert_notmatch('^-- INSERT --', term_getline(buf, 10))})
     call term_sendkeys(buf, ":let [g:autocmd_n, g:autocmd_i] = ['', '']\<CR>")
-    call delete('XTextChangedI3')
+    call writefile([], 'XTextChangedI3')
```

Comment:
call writefile([], 'XTextChangedI3', 'D') ?

Reply:
This will delete the file every time `s:validate_mixed_textchangedi()` returns, which isn't really necessary as it only needs to delete the file at the end of the whole test. I guess it's better to add an explicit `:defer delete()` above.